frontend/src/packages/auth/auth-api.ts

```diff
@@ -16,8 +17,11 @@ type Constructor = {
 };
 
 class AuthApi extends BaseHttpApi {
+  private authStorage: Storage;
```

Comment:
```suggestion
  private storage: Storage;
```

Reply:
I tried making this change and replace authStorage with storage, but it would cause an error: 
`Class 'AuthApi' incorrectly extends base class 'BaseHttpApi'. Types have separate declarations of a private property 'storage'.`
As far as I could understand, it happens because of BaseHttpApi already having its own storage property, the best solution I could come up with is to change the property **storage** in BaseHttpApi from private to protected. And completely remove assignment of private storage in auth-api. But honestly, I am not sure if it is a code idea to change the base class like this: 
<img width="482" alt="Screenshot 2023-08-23 at 12 18 14" src="https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/62558753/e9ce1863-953b-4ace-ad3c-35e1bdb175b1">
<img width="711" alt="Screenshot 2023-08-23 at 12 18 2
<img width="762" alt="Screenshot 2023-08-23 at 12 18 39" src="https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/62558753/ae96cd27-c013-4c12-a63f-570641dccc21">
6" src="https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/62558753/3f4fe82e-90ca-4b8d-94af-f32437b4b5bd">

Reply:
Stay with protected storage one) let's have only one storage

Reply:
Oh, yes, my apologies